src/handlers/swap_handlers.ts

```diff
@@ -233,6 +271,30 @@ export class SwapHandlers {
                 .getPriceComparisonFromQuote(CHAIN_ID, marketSide, quote)
                 ?.map((sc) => priceComparisonUtils.renameNative(sc));
         }
+
+        if (quote.extendedQuoteReport && kafkaProducer) {
+            const bytesPos = quote.data.indexOf('869584cd');
+            // tslint:disable-next-line
+            const quoteId = quote.data.slice(bytesPos + 118, bytesPos + 128);
```

Comment:
hmm, a few more magic numbers. Can we actually extract this logic into a utility function, maybe called `getQuoteIdFromSwapQuote`?
```
function getQuoteIdFromSwapQuote(quote) {
    const bytesPos = quote.data.indexOf('869584cd'); // some comment about where this comes from
    const quoteIdOffset = 118; // some comment about where this comes from
    const startingIndex = bytePos + quoteIdOffset;
    const quoteId = quote.data.slice(startingIndex, startingIndex + 10);
    return quoteId;
}